test/integration/framework.go

```diff
@@ -65,8 +68,22 @@ func withConfigGetFreshApiserverAndClient(
 	t *testing.T,
 	serverConfig *TestServerConfig,
 ) (servicecatalogclient.Interface,
+	*server.ServiceCatalogServerOptions,
```

Comment:
Why do we need to pass the server options back out in all these other cases?

Reply:
We don't *need* to. What's the harm in it?